db.sql

```diff
@@ -0,0 +1,47 @@
+-- Create database
+CREATE DATABASE twd_database;
+
+-- Connect to the database
+\c twd_database;
+
+
+-- Create a 'topics' table
+CREATE TABLE topics (
+    id SERIAL PRIMARY KEY,
```

Comment:
These schemas are close but make them conform to the API Spec

   "title": "string",
      "summary": "string",
      "description": "string",
      "type": "project",
      "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
      "created": "2024-03-09T21:32:16.780Z",
      "updated": "2024-03-09T21:32:16.780Z"

[Spec](tulsawebdevs.org/docs/api)

If that means the front end form they're working on is missing fields let them/us know and add it to the issue.

Reply:
Front end form issue:
https://github.com/tulsawebdevs/website/issues/86